cpp/src/arrow/compute/kernels/vector_sort.cc

```diff
@@ -651,6 +651,11 @@ void AddSortingKernels(VectorKernel base, VectorFunction* func) {
   base.exec = ExecTemplate<UInt64Type, BooleanType>::Exec;
   DCHECK_OK(func->AddKernel(base));
 
+  // duration type
+  base.signature = KernelSignature::Make({InputType::Array(Type::DURATION)}, uint64());
```

Comment:
Might want to update (or remove) the function comment.

Reply:
Sorry, could you clarify? The comment is meant to clarify that this is adding the kernel that handles duration types (despite since there are a lot of types in play and it can be a little unclear which case this is)

Reply:
Sorry, I meant the comment at the top of the function (alternatively, you could just remove this comment)

```
// Sort indices kernels implemented for
//
// * Boolean type
// * Number types
// * Base binary types
```

Reply:
Ah d'oh, thanks. Updated.